dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/UserServiceBean.java

```diff
@@ -302,7 +303,7 @@ private AuthenticatedUser createAuthenticatedUserForView(Object[] dbRowValues, S
         for (Object[] dbResultRow : dbRoleResults) {
 
             String groupIdentifier = UserUtil.getStringOrNull(dbResultRow[0]);
-            String groupRole = UserUtil.getStringOrNull(dbResultRow[1]);
+            String groupRole = RoleTranslationUtil.getNameFromAlias(UserUtil.getStringOrNull(dbResultRow[1]));
```

Comment:
Returning nulls in RoleTranslationUtil.getNameFromAlias actually can break this code. 
There is no 100% chance that role alias will have some translation (case when role is not built in one).
It can lead to situation when there is some role for the user, but we omitting it.

Maybe `RoleTranslationUtil.getNameFromAlias` could have extra parameter for some fallback value?